substrate/frame/system/src/extensions/weights.rs

```diff
@@ -0,0 +1,197 @@
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Autogenerated weights for `frame_system_extensions`
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2023-12-13, STEPS: `2`, REPEAT: `2`, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! WORST CASE MAP SIZE: `1000000`
+//! HOSTNAME: `gleipnir`, CPU: `AMD Ryzen 9 7900X 12-Core Processor`
+//! WASM-EXECUTION: `Compiled`, CHAIN: `Some("dev")`, DB CACHE: 1024
+
+// Executed Command:
+// ./target/debug/node-template
+// benchmark
+// pallet
+// --steps=2
+// --repeat=2
+// --extrinsic=*
+// --wasm-execution=compiled
+// --heap-pages=4096
+// --pallet=frame-system-extensions
+// --chain=dev
+// --output=./substrate/frame/system/src/extensions/weights.rs
+
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+#![allow(missing_docs)]
+
+use frame_support::{traits::Get, weights::{Weight, constants::RocksDbWeight}};
+use core::marker::PhantomData;
+
+/// Weight functions needed for frame_system.
+pub trait WeightInfo {
+	fn check_genesis() -> Weight;
+	fn check_mortality() -> Weight;
+	fn check_non_zero_sender() -> Weight;
+	fn check_nonce() -> Weight;
+	fn check_spec_version() -> Weight;
+	fn check_tx_version() -> Weight;
+	fn check_weight() -> Weight;
+}
+
+/// Weight functions for `frame_system_extensions`.
+pub struct SubstrateWeight<T>(PhantomData<T>);
+impl<T: crate::Config> WeightInfo for SubstrateWeight<T> {
+	/// Storage: `System::BlockHash` (r:1 w:0)
+	/// Proof: `System::BlockHash` (`max_values`: None, `max_size`: Some(44), added: 2519, mode: `MaxEncodedLen`)
+	fn check_genesis() -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `54`
+		//  Estimated: `3509`
+		// Minimum execution time: 33_433_000 picoseconds.
+		Weight::from_parts(35_297_000, 0)
+			.saturating_add(Weight::from_parts(0, 3509))
+			.saturating_add(T::DbWeight::get().reads(1))
+	}
+	/// Storage: `System::BlockHash` (r:1 w:0)
+	/// Proof: `System::BlockHash` (`max_values`: None, `max_size`: Some(44), added: 2519, mode: `MaxEncodedLen`)
+	fn check_mortality() -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `92`
+		//  Estimated: `3509`
+		// Minimum execution time: 59_292_000 picoseconds.
+		Weight::from_parts(61_787_000, 0)
+			.saturating_add(Weight::from_parts(0, 3509))
+			.saturating_add(T::DbWeight::get().reads(1))
+	}
+	fn check_non_zero_sender() -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `0`
+		//  Estimated: `0`
+		// Minimum execution time: 7_283_000 picoseconds.
+		Weight::from_parts(8_115_000, 0)
+			.saturating_add(Weight::from_parts(0, 0))
+	}
+	fn check_nonce() -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `0`
+		//  Estimated: `0`
+		// Minimum execution time: 41_448_000 picoseconds.
+		Weight::from_parts(42_500_000, 0)
```

Comment:
It should read the account data (from the Account map), so maybe something wrong with benchmarks code?

Reply:
`pallet_transaction_payment` now has real weights.